### ISSUE TITLE
After doctrine orm 2.13 Access to offset XY on an unknown class Doctrine\\ORM\\Mapping\\AssociationMapping.

### ISSUE THREAD

**Issue by mmarton** [phpstan/phpstan-doctrine#358]
Hi!

I've encountered the following - possible false positive - phpstan error after upgrading to doctrine orm 2.13

### env:
- php: 8.1
- doctrine/orm: 2.13.1
- doctrine/dbal: 3.4
- phpstan/phpstan: 1.8.2
- phpstan/phpstan-doctrine: 1.3.11

### code:
```php
<?php

declare(strict_types=1);

namespace App\Resolver;

use Doctrine\ORM\EntityManagerInterface;

class CollectionResolver
{
    public function __construct(
         private readonly EntityManagerInterface $entityManager
    ) {
    }

    public function resolve(string $className, string $propertyName, mixed $value): string
    {
        $propertyInfo = $this->entityManager
            ->getClassMetadata($className)
            ->getAssociationMapping($propertyName);

        if ($propertyInfo['targetEntity'] === MyEntity::class) { // error for this line
            // ...
        }
    }
}
```

### Expected
 no error
 
 ### Actual
  **Access to offset 'targetEntity' on an unknown class Doctrine\ORM\Mapping\AssociationMapping.**

**Comment by ondrejmirtes** [comment]
Please try phpstan-doctrine 1.3.x-dev.

**Comment by mmarton** [comment]
It works as expected with 1.3.x-dev. Thanks

**Comment by ondrejmirtes** [comment]
I'll release it soon-ish 😊

**Comment by ondrejmirtes** [comment]
Released: https://github.com/phpstan/phpstan-doctrine/releases/tag/1.3.12
